src/EnergyPlus/DXCoils.cc

```diff
@@ -2979,895 +2979,978 @@ void GetDXCoils(EnergyPlusData &state)
+                            checkCurveIsNormalizedToOne(state,
+                                                        std::string{RoutineName} + CurrentModuleObject,
+                                                        thisDXCoil.Name,
+                                                        thisDXCoil.HCapFTemp,
+                                                        cFieldName,
+                                                        fieldValue,
+                                                        InletAirTemp);
+                        } else {
+                            checkCurveIsNormalizedToOne(state,
+                                                        std::string{RoutineName} + CurrentModuleObject,
+                                                        thisDXCoil.Name,
+                                                        thisDXCoil.HCapFTemp,
+                                                        cFieldName,
+                                                        fieldValue,
+                                                        InletAirTemp,
+                                                        InletWaterTemp);
+                        }
+                    }
                 }
             }
-        }
 
-        if (!lAlphaBlanks(14)) {
-            thisDXCoil.HCOPFTemp = GetCurveIndex(state, Alphas(14));
-            if (thisDXCoil.HCOPFTemp == 0) {
-                ShowSevereError(state, format("{}{}=\"{}\", invalid", RoutineName, CurrentModuleObject, thisDXCoil.Name));
-                ShowContinueError(state, format("...not found {}=\"{}\".", cAlphaFields(14), Alphas(14)));
-                ErrorsFound = true;
-            } else {
-                // Verify Curve Object, only legal types are BiQuadratic or Cubic
-                ErrorsFound |= Curve::CheckCurveDims(state,
-                                                     thisDXCoil.HCOPFTemp, // Curve index
-                                                     {1, 2},               // Valid dimensions  // MULTIPLECURVEDIMS
-                                                     RoutineName,          // Routine name
-                                                     CurrentModuleObject,  // Object Type
-                                                     thisDXCoil.Name,      // Object Name
-                                                     cAlphaFields(14));    // Field Name
+            cFieldName = "Heating Capacity Function of Air Flow Fraction Curve Name"; // Alphas(12)
+            fieldValue = s_ip->getAlphaFieldValue(fields, schemaProps, "heating_capacity_function_of_air_flow_fraction_curve_name");
+            if (!fieldValue.empty()) {
+                thisDXCoil.HCapFAirFlow = GetCurveIndex(state, fieldValue);
+                if (thisDXCoil.HCapFAirFlow == 0) {
+                    ShowSevereError(state, format("{}{}=\"{}\", invalid", RoutineName, CurrentModuleObject, thisDXCoil.Name));
```

Comment:
Another `ShowSevereItemNotFound`. See `UtilityRoutines` for wrappers for the common errors, e.g., `EmptyField`, `InvalidKey`, `DuplicateName`, `BadMin`.